src/components/App/App.js

```diff
@@ -3,25 +3,25 @@ import { Provider } from 'react-redux';
 import { CircleArrow as ScrollUpButton } from 'react-scroll-up-button';
 
 import Test from '../Test';
-import Chat from '../Chat';
+import Chat from '../chat';
 
-import { GraphqlServiceProvider } from '../../graphqlServiceContext';
-import { graphqlService } from '../../services';
+import { ApolloServiceProvider } from '../../services/ApolloService/ApolloServiceContext';
+import { client } from '../../services/ApolloService/ApolloService';
 import store from '../../store/store';
 
 // import Routes from '../routes';
 
 const App = () => (
   <Provider store={store}>
-    <GraphqlServiceProvider value={graphqlService}>
+    <ApolloServiceProvider value={client}>
```

Comment:
I don't think that we need to use `ApolloServiceProvider`. We are going to dispatch actions by ourselves and handle them in sagas. The only thing that we need from `apollo` is actual client.

Reply:
Deleted ApolloProvider